OneSignalSDK/onesignal/core/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="NoToolBarTheme">
+        <item name="android:windowAnimationStyle">@android:style/Animation</item>
+        <item name="android:windowBackground">@android:color/transparent</item>
+        <item name="android:windowIsTranslucent">true</item>
+        <item name="windowNoTitle">true</item>
+        <item name="windowActionBar">false</item>
+    </style>
+</resources>
```

Comment:
This doesn't seem to fix the problem, in the example app it changes the background color of the status bar, confirmed on an Android 14 emulator.

If this is not easy to fix we can do this in a different PR, so not block this one or a release.